gradle.properties

```diff
@@ -1 +1 @@
-org.gradle.jvmargs=-Xmx2560M
+org.gradle.jvmargs=-Xmx5120M
```

Comment:
I don't think you meant to commit this. And this change could be problematic for people with only 8GB of ram.

Reply:
Agreed. I only have four gigs to spare, and I’m pretty sure a chunk of people do as well

Reply:
I added this because on one of my previous prs, it was failing due to a lack of memory.